nix/stack-packages.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
As we are doing the overriding via through the module in `pkgs.nix` now, we don't really need this file anymore.